verticaltabs.js

```diff
@@ -136,8 +136,8 @@ VerticalTabs.prototype = {
     this.inferFromText = window.ToolbarIconColor.inferFromText;
     this.receiveMessage = window.gBrowser.receiveMessage;
 
-    let OldPrintPreviewListenerEnter = window.PrintPreviewListener.onEnter;
-    let OldPrintPreviewListenerExit = window.PrintPreviewListener.onExit;
+    let OldPrintPreviewListenerEnter = window.PrintPreviewListener.onEnter.bind(window.PrintPreviewListener);
+    let OldPrintPreviewListenerExit = window.PrintPreviewListener.onExit.bind(window.PrintPreviewListener);
 
     window.PrintPreviewListener.onEnter = () => {
```

Comment:
Say, shouldn't we put these back to their previous definitions when we exit or on tabs-on-top?